src/components/templates/PartyMap/components/Map/Map.tsx

```diff
@@ -69,11 +70,14 @@ export const Map: React.FC<MapProps> = ({
   );
   const rowsArray = useMemo(() => Array.from(Array(totalRows)), [totalRows]);
 
-  const [mapBackgroundUrl] = useMapBackground(venue?.mapBackgroundImageUrl);
+  const [mapBackground] = useCheckImageWithFallback(
+    venue?.mapBackgroundImageUrl,
+    DEFAULT_MAP_BACKGROUND
+  );
 
   useEffect(() => {
     const img = new Image();
-    img.src = mapBackgroundUrl;
+    img.src = mapBackground ?? DEFAULT_MAP_BACKGROUND;
```

Comment:
why do we need a fallback here too? isn't `mapBackground` with fallback?

Reply:
`useCheckImageWithFallback` guards from invalid image. But it still needs to return something while the image is loading. 

For example, in the `Map.tsx` component the map background loads for a split second. While it's loading nothing is displayed. If I change the return value of `useCheckImageWithFallback` to be `string` rather than `string | undefined` I must return something while the image is loading. This introduces _flickering_ where for this split second `DEFAULT_MAP_BACKGROUND` is shown. That doesn't look nice IMO so I tried to preserve the original behavior.

Also, to improve readability I renamed the hook to be `useValidImage` and the param to `fallbackIfInvalid`.
And finally, the type was set correctly `string | undefined` but I was still returning `src ?? fallbackSrc` which resulted in flickering (forgot to fix that).
Renamed and fixed here https://github.com/sparkletown/sparkle/pull/2025/commits/5f89f4dfa98705fd30dad433d7627dd14c1c9057